handlers/whatsapp_legacy/handler_test.go

```diff
@@ -1070,21 +1070,6 @@ var mediaCacheSendTestCases = []OutgoingTestCase{
 	},
 }
 
-var hsmSupportSendTestCases = []OutgoingTestCase{
```

Comment:
hsm type was replaced by template, so removing support for that as that is no longer used